examples/cra-kitchen-sink/src/enzyme.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
We have this setup on root level, no need to duplicate it here

Reply:
same as https://github.com/storybooks/storybook/pull/2414#discussion_r154497265